src/components/home/useful-links/useful-links.tsx

```diff
@@ -0,0 +1,37 @@
+/*
+ * This file is part of NER's PM Dashboard and licensed under GNU AGPLv3.
+ * See the LICENSE file in the repository root folder for details.
+ */
+
+import styles from './useful-links.module.css';
+
+const UsefulLinks: React.FC = () => {
+  return (
+    <div>
+      <h2>Useful links</h2>
+      <h3>Finance</h3>
+      <ul>
+        <li>
+          <a className={styles.link} href="https://twitter.com/" target="_blank" rel="noreferrer">
```

Comment:
I believe we have a link pipe that can be used to abstract the creation of an anchor tag

Reply:
Actual link: https://docs.google.com/document/d/1M5Ldy9L1BifBo18tdKpv3CH-frRneyEK26hUXbtMg7Q/edit